ow_sim_tests/scripts/test_camera_set_exposure.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python
+
+# The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+# Research and Simulation can be found in README.md in the root directory of
+# this repository.
+
+import rospy
+import roslib
+import unittest
+import owl_msgs.msg
+
+from action_testing import *
+
+PKG = 'ow_sim_tests'
+TEST_NAME = 'camera_set_exposure'
+roslib.load_manifest(PKG)
+
+class TestCameraSetExposure(unittest.TestCase):
+
+  @classmethod
+  def setUpClass(cls):
+    rospy.init_node("test_camera_set_exposure")
+
+    set_ignore_action_checks(True)
+    # proceed with test only when ros clock has been initialized
+    while rospy.get_time() == 0:
+      rospy.sleep(0.1)
+  
+  def test_01_camera_set_exposure(self):
+    camera_set_exposure_result = test_action(self,
+      'CameraSetExposure', owl_msgs.msg.CameraSetExposureAction,
+      owl_msgs.msg.CameraSetExposureGoal(
+        automatic = True,
```

Comment:
Actually, the `automatic` field is completely ignored, in both the PLEXIL interface and the action script.  It shouldn't make a difference what it is set to, though I agree False is the proper value.  I think we should put some comments in the implementation that this field is ignored (as a separate task).

Reply:
Sending `automatic = true` to  `CameraSetExposure`'s action server actually results in an action aborted.
```
[ERROR:/lander_action_servers] CameraSetExposure: Aborted - CameraSetExposure action does not support autoexposure.
```
Perhaps it ought not work this way. Mind you it is impossible to get this result by calling the action client script because it simply does not expose the option, it is perfectly possible to arrive at this result by calling the action from C++ code, Python code, or by publishing directly to the topic `CameraSetExposure/goal`.

Reply:
This is why the `test_camera_set_exposure.test` fails.